scripts/extract-elements.js

```diff
@@ -69,6 +73,19 @@ request(refUrl, (error, response, html) => {
     const $ = cheerio.load(html);
     const elements = extractElements($);
     if (elements.length !== expectedElCount) {
+        try {
+            const prevEls = fs.readFileSync(dataPath, 'utf8').split('\n');
+            const added = elements.filter(n => prevEls.indexOf(n) === -1);
+            const removed = prevEls.filter(n => elements.indexOf(n) === -1);
+
+            throw new Error(
+                'Found extra elements: [' + added.join(',') +
+                '] and did not find elements: [' + removed.join(',') + ']'
```

Comment:
Don't know the context for this but the message can be improved. If I understand correctly we want to convey the following, right?

Elements not observed before = added
Elements observed before but missing in current session = removed.

Reply:
good call, clarified in https://github.com/plotly/dash-html-components/pull/194/commits/0ce630da4ab4e075259633c0a641a5d92c7d698a